assets/src/components/TableAssignment.js

```diff
@@ -125,14 +125,22 @@ function CustomAssignmentTable (props) {
 
   useEffect(() => {
     if (currentWeekRow.current) {
-      const tableHeaderOffset = 56
+      const tableHeaderOffset = 50
       tableRef.current.parentNode.scrollTo({
-        top: currentWeekRow.current.offsetTop - tableHeaderOffset,
+        top: currentWeekRow.current.offsetTop-tableHeaderOffset,
         behavior: 'smooth'
       })
     }
   })
 
+  const isItFirstCurrentWeekIndicator= i => {
```

Comment:
This logic only adds the `rootRef` for the first row when the week property is actually visible. With this, I see that the scroll is rolling to a currect position